ert_shared/storage/connection.py

```diff
@@ -58,3 +58,16 @@ def set_global_info(project_path: Union[str, Path]):
     if path.is_symlink():
         path.unlink()
     path.symlink_to(_json_path(project_path))
+
+
+def get_project_id() -> Path:
+    """Return a path to `storage_server.json`."""
+
+    if "XDG_RUNTIME_DIR" in os.environ:
+        project_path = Path(os.environ["XDG_RUNTIME_DIR"]) / "ert"
+    else:
+        project_path = Path(f"/tmp/ert-{getpass.getuser()}")
+    json_path = project_path / "storage_server.json"
```

Comment:
```os.path.join(...)``` ?

Reply:
we should use `pathlib`, see e.g. https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/